### PR TITLE
Fix aws-go-eks

### LINKED ISSUE #1212: aws-go-eks is failing
Update to use correct dependency paths, and update other deps while there.

### PR DESCRIPTION
**Pull request by jkisk**
Fixes #1212

### CODE REVIEW

File: aws-go-eks/main.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	appsv1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/apps/v1"
 	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/core/v1"
 	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/meta/v1"
-	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/providers"
+	provider "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes"
```
Comment (lblackstone) [inline aws-go-eks/main.go:12]:
```suggestion
	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes"
```

File: aws-go-eks/main.go [hunk 2]
```diff
@@ -143,7 +143,7 @@ func main() {
 		ctx.Export("kubeconfig", generateKubeconfig(eksCluster.Endpoint,
 			eksCluster.CertificateAuthority.Data().Elem(), eksCluster.Name))
 
-		k8sProvider, err := providers.NewProvider(ctx, "k8sprovider", &providers.ProviderArgs{
+		k8sProvider, err := provider.NewProvider(ctx, "k8sprovider", &provider.ProviderArgs{
```
Comment (lblackstone) [inline aws-go-eks/main.go:146]:
```suggestion
		k8sProvider, err := kubernetes.NewProvider(ctx, "k8sprovider", &kubernetes.ProviderArgs{
```

File: aws-go-eks/README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ This example creates an AWS EKS Cluster and deploys a sample container applicati
 ### Prerequisites
 
 1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
-2. [Install Go 1.13](https://golang.org/doc/install)
+2. [Install Go 1.16](https://golang.org/doc/install)
```
Comment (lblackstone) [inline aws-go-eks/README.md:11]:
```suggestion
2. [Install Go](https://golang.org/doc/install)
```

**Review by lblackstone — APPROVED** [review]
Just a few minor suggestions. Looks good overall!

**Comment by jkisk** [comment]
Thanks Levi!

File: aws-go-eks/go.mod [hunk 4]
```diff
@@ -3,7 +3,7 @@ module aws-go-eks
 go 1.16
```
Comment (viveklak) [inline aws-go-eks/go.mod:3]: Should we switch to 1.17? I guess pulumi-eks is still on 1.16 so this limits it?

Comment (jkisk) [reply aws-go-eks/go.mod:3]: Yep, exactly.

**Pull request merged**